Userland/Libraries/LibWeb/CSS/CSS.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021, Sam Atkins <atkinssj@serenityos.org>
+ * Copyright (c) 2023, Tim Flynn <trflynn89@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+#include <AK/StringView.h>
+#include <LibJS/Forward.h>
+#include <LibWeb/WebIDL/ExceptionOr.h>
+
+// https://www.w3.org/TR/cssom-1/#namespacedef-css
+namespace Web::CSS {
+
+WebIDL::ExceptionOr<String> escape(JS::VM&, StringView identifier);
+
+bool supports(JS::VM&, StringView property, StringView value);
+WebIDL::ExceptionOr<bool> supports(JS::VM&, StringView condition_text);
```

Comment:
passing `StringView`s to IDL-generated function is not something we've usually done before, usually it's `{Deprecated,}String const&`. Seems like something we can start doing at scale, just wanted to point that out.

Reply:
Yeah, the caller has a `DeprecatedString` (by way of CSSOMString), and these functions don't want ownership of the strings at all, so I figured one less thing to have to port to `String`